Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -63,7 +64,12 @@ struct SwiftUIPaymentButtonDemo: View {
                     selectedEdge = PaymentButtonEdges.allCases[edgesIndex]
                     buttonId += 1
                 }
-
+                Stepper("Custom Corner Radius: \(customEdge)", value: $customEdge, in: 0...100).onChange(of: customEdge) { _ in
+                    if selectedEdge.description == "custom" {
+                        selectedEdge = PaymentButtonEdges.custom(CGFloat(customEdge))
+                        buttonId += 1
```

Comment:
curious q: is button ID used by SwiftUI to re-render?

Reply:
```suggestion
                        buttonID += 1
```
Also side note our team uses `ID` naming convention for iOS.

Reply:
> curious q: is button ID used by SwiftUI to re-render?

Yeah the views use the `buttonId` as their `.id()` and it triggers a re-render 

> Also side note our team uses ID naming convention for iOS.
It was already declared I can change it though